cupy/_logic/truth.py

```diff
@@ -4,6 +4,14 @@
 from cupy import _util
 
 
+setxorkernel = cupy._core.ElementwiseKernel(
```

Comment:
Could you prefix the kernel name with an underscore?

```suggestion
_setxorkernel = cupy._core.ElementwiseKernel(
```